### ISSUE TITLE
i install it, but can't work!

### ISSUE THREAD

**Issue by lxd235** [omarish/django-cprofile-middleware#20]
#### description
1. here is my log in /var/log/aio/message;
2. django version is 1.8.10
#### log content:
```
May  4 16:48:52 localhost uwsgi: Traceback (most recent call last):
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django/core/handlers/wsgi.py", line 170, in __call__
May  4 16:48:52 localhost uwsgi: self.load_middleware()
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django/core/handlers/base.py", line 50, in load_middleware
May  4 16:48:52 localhost uwsgi: mw_class = import_string(middleware_path)
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django/utils/module_loading.py", line 26, in import_string
May  4 16:48:52 localhost uwsgi: module = import_module(module_path)
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/importlib/__init__.py", line 109, in import_module
May  4 16:48:52 localhost uwsgi: return _bootstrap._gcd_import(name[level:], package, level)
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 1129, in _exec
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 1471, in exec_module
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django_cprofile_middleware/middleware.py", line 14, in <module>
May  4 16:48:52 localhost uwsgi: from django.utils.deprecation import MiddlewareMixin
May  4 16:48:52 localhost uwsgi: ImportError: cannot import name 'MiddlewareMixin'
May  4 16:48:52 localhost uwsgi: [pid: 6550|app: 0|req: 22/22] 172.16.6.20 () {44 vars in 837 bytes} [Fri May  4 16:48:52 2018] GET /xdashboard/home/?prof => generated 0 bytes in 9 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 42)
May  4 16:48:52 localhost uwsgi: Traceback (most recent call last):
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django/core/handlers/wsgi.py", line 170, in __call__
May  4 16:48:52 localhost uwsgi: self.load_middleware()
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django/core/handlers/base.py", line 50, in load_middleware
May  4 16:48:52 localhost uwsgi: mw_class = import_string(middleware_path)
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django/utils/module_loading.py", line 26, in import_string
May  4 16:48:52 localhost uwsgi: module = import_module(module_path)
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/importlib/__init__.py", line 109, in import_module
May  4 16:48:52 localhost uwsgi: return _bootstrap._gcd_import(name[level:], package, level)
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 1129, in _exec
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 1471, in exec_module
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django_cprofile_middleware/middleware.py", line 14, in <module>
May  4 16:48:52 localhost uwsgi: from django.utils.deprecation import MiddlewareMixin
May  4 16:48:52 localhost uwsgi: ImportError: cannot import name 'MiddlewareMixin'
May  4 16:48:52 localhost uwsgi: [pid: 6550|app: 0|req: 23/23] 172.16.6.20 () {42 vars in 768 bytes} [Fri May  4 16:48:52 2018] GET /nginx-logo.png => generated 0 bytes in 5 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 46)
May  4 16:48:52 localhost uwsgi: Traceback (most recent call last):
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django/core/handlers/wsgi.py", line 170, in __call__
May  4 16:48:52 localhost uwsgi: self.load_middleware()
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django/core/handlers/base.py", line 50, in load_middleware
May  4 16:48:52 localhost uwsgi: mw_class = import_string(middleware_path)
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django/utils/module_loading.py", line 26, in import_string
May  4 16:48:52 localhost uwsgi: module = import_module(module_path)
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/importlib/__init__.py", line 109, in import_module
May  4 16:48:52 localhost uwsgi: return _bootstrap._gcd_import(name[level:], package, level)
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 1129, in _exec
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 1471, in exec_module
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django_cprofile_middleware/middleware.py", line 14, in <module>
May  4 16:48:52 localhost uwsgi: from django.utils.deprecation import MiddlewareMixin
May  4 16:48:52 localhost uwsgi: ImportError: cannot import name 'MiddlewareMixin'
May  4 16:48:52 localhost uwsgi: [pid: 6550|app: 0|req: 24/24] 172.16.6.20 () {42 vars in 766 bytes} [Fri May  4 16:48:52 2018] GET /poweredby.png => generated 0 bytes in 5 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 11)
May  4 16:48:52 localhost uwsgi: Traceback (most recent call last):
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django/core/handlers/wsgi.py", line 170, in __call__
May  4 16:48:52 localhost uwsgi: self.load_middleware()
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django/core/handlers/base.py", line 50, in load_middleware
May  4 16:48:52 localhost uwsgi: mw_class = import_string(middleware_path)
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django/utils/module_loading.py", line 26, in import_string
May  4 16:48:52 localhost uwsgi: module = import_module(module_path)
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/importlib/__init__.py", line 109, in import_module
May  4 16:48:52 localhost uwsgi: return _bootstrap._gcd_import(name[level:], package, level)
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 1129, in _exec
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 1471, in exec_module
May  4 16:48:52 localhost uwsgi: File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
May  4 16:48:52 localhost uwsgi: File "/root/.pyenv/versions/3.4.4/lib/python3.4/site-packages/django_cprofile_middleware/middleware.py", line 14, in <module>
May  4 16:48:52 localhost uwsgi: from django.utils.deprecation import MiddlewareMixin
May  4 16:48:52 localhost uwsgi: ImportError: cannot import name 'MiddlewareMixin'
May  4 16:48:52 localhost uwsgi: [pid: 6550|app: 0|req: 25/25] 172.16.6.20 () {46 vars in 815 bytes} [Fri May  4 16:48:52 2018] GET /favicon.ico => generated 0 bytes in 7 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 1)

```

**Comment by chimeno** [comment]
Duplicate of #15 .
It seems new release has to be made and upload it.

**Comment by omarish** [comment]
Sorry for the delay. Should be fixed in v1.0.1.
